### PR TITLE
[Hotfix] 0.2.19

### PR DESCRIPTION
**Pull request by perruche**
## FIx

- Fix `AbstractScrollAnimation` by removing `playRange` assignement
Component was throwing an error cause it has FreezedOptions: `Uncaught (in promise) TypeError: Cannot assign to read only property 'playRange' of object '#<Object>'` 

<a href="https://gitpod.io/#https://github.com/studiometa/ui/pull/92"></a>

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "studiometa/ui",
-  "version": "0.2.17",
+  "version": "0.2.19",
```
Comment (titouanmathis) [inline composer.json:3]: **issue:** The previous version was `0.2.18`, you might need to re-fetch master before creating the hotfix 

**Small tip:** you can use `npm version patch` at the root of the project to update all version numbers in one command. Just make sure to disable tagging before running the command:

```sh
npm config -g set git-tag-version=false
npm version patch
```

**Review by titouanmathis — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (perruche) [reply composer.json:3]: it seems 0.2.18 has only been pushed to `develop` not `master`. The hotfix branch does not exist on origin, so maybe it's easier if you can push @KeziahMoselle ?

Comment (KeziahMoselle) [reply composer.json:3]: Sorry, it should be fixed now

Comment (perruche) [reply composer.json]: no worries 😄

**Comment by perruche** [comment]
is it good to merge ?

File: composer.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "studiometa/ui",
-  "version": "0.2.18",
+  "version": "0.2.20",
```
Comment (titouanmathis) [inline composer.json:3]: **issue:** The version should be `0.2.19` here 

I forgot a parameter with the NPM command I gave you @perruche, here is the one to use:

```sh
npm version -ws --include-workspace-root <version>
```

You can run it with `0.2.19` as parameter to make sure all version numbers are correct:

```sh
npm version -ws --include-workspace-root 0.2.19
```

**Review by titouanmathis — CHANGES_REQUESTED** [review]
A small fix to the version numbers, and we are good to go!

**Review by titouanmathis — APPROVED** [review]
🚀

**Pull request merged**